scripts/v1alphaX/delete_cluster.sh

```diff
@@ -5,11 +5,16 @@ METAL3_DIR="$(dirname "$(readlink -f "${0}")")/../.."
 # shellcheck disable=SC1090
 source "${METAL3_DIR}/lib/common.sh"
 
+if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+  V1ALPHAX_CR_PATH=${V1ALPHA3_CR_PATH}
+else
+  V1ALPHAX_CR_PATH=${V1ALPHA2_CR_PATH}
+fi
+
```

Comment:
This could be in lib/common.sh to avoid code duplication

Reply:
done